tools/Google.Cloud.Tools.ReleaseManager/AddCommand.cs

```diff
@@ -68,7 +68,9 @@ protected override void ExecuteImpl(string[] args)
                 Type = ApiType.Grpc,
                 Generator = GeneratorType.Micro,
                 // Let's not include test dependencies, which are rarely useful.
-                TestDependencies = null
+                TestDependencies = null,
+                // Translate the host name into the "short name", e.g. bigquery.googleapis.com => bigquery
+                ShortName = targetApi.HostName.Split('.').First()
```

Comment:
You need to account for the regional default endpoints as well, which are supposed to become more common in the short term.
See https://github.com/googleapis/gapic-generator-csharp/blob/543fe7042d6de918f9bab841256400b1b97b238e/Google.Api.Generator/Generation/ServiceDetails.cs#L60

Reply:
I don't think so - that's based on the default host in a .proto service; this is the "name" from the service config.